src/service/subFolderElementHandler.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+const StandardHandler = require('./standardHandler')
+const mc = require('../utils/metadataConstants')
+const path = require('path')
+
+class SubFolderElementHandler extends StandardHandler {
+  handleDeletion() {
+    this._fillPackage(this.diffs.destructiveChanges)
+  }
+
+  handleAddition() {
+    super.handleAddition()
+    if (!this.config.generateDelta) return
+
+    const fileName = this.splittedLine
+      .slice(this.splittedLine.length - 1)
+      .join(path.sep)
+
+    this._copyFiles(
+      path.join(this.config.repo, fileName),
+      path.join(this.config.output, fileName)
+    )
+  }
```

Comment:
Maybe in this case we should not call the parent
And deal with the -meta file ourself
```
if (this.metadata[this.type].metaFile === true) {
  this._copyFiles(path.join(this.config.repo, fileName + mc.METAFILE_SUFFIX), path.join(this.config.output, fileName + mc.METAFILE_SUFFIX))
}
```
Something like that